blog/2021-12-20-openhab-3-1-release.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: BlogPost
+category: blog
+title: openHAB 3.2 Release
+author: Kai Kreuzer
+date: '2021-12-20T00:00:00+01:00'
+previewimage: /uploads/2021-06_summer.jpg
```

Comment:
I'll try to come up with a nice new image, if you wish.

Reply:
I have one actually, i'll post soon